distribution/tools/plugin-cli/src/test/java/org/opensearch/plugins/InstallPluginCommandTests.java

```diff
@@ -1049,7 +1049,7 @@ public void assertInstallPluginFromUrl(
 
     public void testOfficialPlugin() throws Exception {
         String url = "https://artifacts.opensearch.org/releases/plugins/analysis-icu/"
-            + Version.CURRENT
+            + Build.CURRENT.getQualifiedVersion()
```

Comment:
@saratvemulapalli sorry, as far as I can tell, `Version.CURRENT == Build.CURRENT.getQualifiedVersion()` (at least, should be), why we are changing that?

Reply:
Answering my own question: `Version.CURRENT == Build.CURRENT.getQualifiedVersion()` is only in tests, otherwise taken from the JAR artifact ...

Reply:
Thank you @reta for the review.
`Version.CURRENT` doesnt include the qualifier where as `Build.CURRENT.getQualifiedVersion()` does. 
This helps us structuring artifacts in dedicated folders for releases like `alpha1`, `beta1`, `rc1`